server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -134,6 +134,7 @@ func (p *Deployer) startPRRevisionWorkflow(ctx workflow.Context, deployment terr
 		ParentClosePolicy: enums.PARENT_CLOSE_POLICY_ABANDON,
 	})
 
+	// TODO: Pass cut off days in the req
```

Comment:
?

Reply:
I'll add this in a follow up PR that sets up the temporal worker and reads this config from the global config